GitExtensions.ruleset

```diff
@@ -0,0 +1,177 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Git Extensions Rules" Description="Hand selected set of analyzer rules for use within the GitExtensions project." ToolsVersion="15.0">
+  <Include Path="minimumrecommendedrules.ruleset" Action="Default" />
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA1301" Action="None" />
+    <Rule Id="CA2101" Action="None" />
+  </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA0001" Action="None" />
```

Comment:
:memo: SA0001 is fine to disable here to minimize the changes in this PR, but removing this line should be the first change for the next PR. The warning links to documentation for SA0001 which described the process for getting this rule corrected.